cluster.yml

```diff
@@ -51,13 +51,13 @@
   any_errors_fatal: "{{ any_errors_fatal | default(true) }}"
   roles:
     - { role: kubespray-defaults}
-    - { role: etcd, tags: etcd, etcd_cluster_setup: true }
```

Comment:
This setup var is really important and I think you misunderstand what it does. It's meant to allow us to re-run the role again for k8s-cluster, but skip etcd setup since it takes a long time for ansible to skip all those tasks for non-etcd nodes. This is more noticeable in large clusters.

Please leave it in.

Reply:
So you'd rather change the var to false in the cluster.yml file, than in the k8s-cluster.yml with configuration changes?
If you set it to false in k8s-cluster or all.yml, the role will be skipped in a re-run. This change simply just moved it from the playbook it self, but still honoring it in runs / setup.

Also when the PR with a seperate cluster for etcd events was merged, a new var was added and not honored here too.

The use of changing vars in playbooks seems odd and undocumented, whereas vars in kubespray is in general in the k8s-cluster.yml file.

The new playbook here also does not run etcd cluster related stuff on k8s nodes but only on etcd nodes, causing this PR to be 4x faster.

Reply:
etcd role gets evaluated for k8s nodes so it can sync certs. Let me try to run this in the fake scale mode and see how it performs.

Reply:
Yes. That still happens here:

https://github.com/woopstar/kubespray/blob/86e3506ae637e2050f953cd85cda5ca16a12ce6f/cluster.yml#L56